README.adoc

```diff
@@ -1,3 +1,24 @@
+= C++ example project manually scanned on SonarQube
+
+It is very easy to manually analyze a C/C++/Objective-C project with SonarQube:
+
+. Create a `sonar-project.properties` file to store your configuration
+.. Add a projectKey of your project
+.. Add a Build Wrapper output directory
+.. https://docs.sonarqube.org/latest/user-guide/user-token/[Generate a unique token] and set it as "sonar.login"
+.. Set `sonar.host.url` to your SonarQube server URL
+. Download and unzip the Build Wrapper for Linux from {SonarQube URL}/static/cpp/build-wrapper-linux-x86.zip
+. Add the Build Wrapper to your `$PATH`
+. Download and unzip the https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/[SonarScanner] for Linux
+. Add the bin directory of the SonarScanner to your `$PATH`
+. Wrap your clean build command with the Build Wrapper. E.g.: `build-wrapper-linux-x86-64 --out-dir bw-output make clean all` (with `bw-output` being the same directory that the one set in `sonar-project.properties`)
```

Comment:
```suggestion
. Wrap your clean build command with the BuildWrapper. E.g.: `build-wrapper-linux-x86-64 --out-dir bw-output make clean all` (with `bw-output` being the same directory as the one set in `sonar-project.properties`)
```
"BuildWrapper" for consistency with "SonarScanner"

Reply:
I prefer being consistent with SQ documentation that uses "SonarScanner" but "Build Wrapper" if you don't mind